pino.js

```diff
@@ -170,7 +184,7 @@ function Pino (opts, stream) {
 }
 
 Pino.prototype = new EventEmitter()
-defineLevelsProperty(Pino.prototype)
+defineLevelsProperty(Pino.prototype, defaultLevelsValues)
```

Comment:
Yes, less code changes, _but_ with your solution you __always__ have to call `defineLevelsProperty` in the `Pino` constructor. For every child. Even if you only use the standard levels. In my solution it's only done when needed, and instead I rely on prototype chaining.